docs/guides/issue_triage.md

```diff
@@ -0,0 +1,135 @@
+# Project issues
+
+General issues are listed at
+<https://github.com/project-chip/connectedhomeip/issues> .
+
+In order to be able to effectively follow up on these issues, they are separated
+into groups for further review and fixing based on current maintainers on
+specific areas of the code. The separation is done via `labels`.
+
+Issues that still need triaging are labeled as `needs triage`:
+
+<https://github.com/project-chip/connectedhomeip/issues?q=is%3Aissue+is%3Aopen+label%3A%22needs+triage%22>
+
+## Non-SDK issues
+
+In some cases the issue is integrating with ecosystems or environments are not
+supported. In these cases, issues can be re-directed to specific support pages
+rather than the SDK GitHub issues.
+
+### Google Integration
+
+- General integration link: <https://developers.home.google.com/matter>
+- Support via the communities areas:
+  - Stack overflow:
+        <https://stackoverflow.com/questions/tagged/google-smart-home>
+  - Google Nest Community:
+        <https://www.googlenestcommunity.com/t5/Smart-Home-Developer-Forum/bd-p/Smart-Home-Developer-Forum>
+
```

Comment:
Would be good to ping Woody and Tom about darwin / samsung support pages. Does amazon come up much in these kind of issues?

Reply:
Not a blocker for this IMO, more info can easily be added later.

Reply:
@woody-apple @bzbarsky-apple could you help me with what to write for the Darwin sections here?
@arkq do you have samsung support details here (I don't think I have the github handle for Tom Manly here ...)
@chrisdecenzo do we have Amazon guides here for amazon integration support issues?

Reply:
Thank you. Added an apple integration section.